collector/tcpstat_linux.go

```diff
@@ -75,6 +84,11 @@ func NewTCPStatCollector(logger log.Logger) (Collector, error) {
 			"Number of connection states.",
 			[]string{"state"}, nil,
 		), prometheus.GaugeValue},
+		descPort: typedDesc{prometheus.NewDesc(
```

Comment:
This should probably not be a separate metric, but rather change the labels for the existing metric depending on the flag.

Reply:
I'm not sure how to implement this. The new metric is working per source/dest port, that is being described by labels "type" and "port". Based on your comment, the solution should be one metric that based on the flags should contain the labels or not? The prometheus library won't allow me to do so. Can you please explain further, how you mean it? thanks

Reply:
@SuperQ Hi, can I ask you about the implementation? We need to have the metrics and I would like to finish the MR, thanks:)

Reply:
I think what @SuperQ suggests is adding these as labels to exisiting connection_states metric. We'd need to add the total number there as well though, maybe using 'port=0' label for the remainder